opal/corelib/runtime.js

```diff
@@ -1189,19 +1189,23 @@
     var subscriber, subscribers = Opal.stub_subscribers,
         i, ilength = stubs.length,
         j, jlength = subscribers.length,
-        method_name, stub;
+        method_name, stub,
+        opal_stubs = Opal.stubs;
 
     for (i = 0; i < ilength; i++) {
       method_name = stubs[i];
-      // Save method name to populate other subscribers with this stub
-      Opal.stubs[method_name] = true;
-      stub = Opal.stub_for(method_name);
 
-      for (j = 0; j < jlength; j++) {
-        subscriber = subscribers[j];
+      if(!opal_stubs.hasOwnProperty(method_name)) {
+        // Save method name to populate other subscribers with this stub
+        opal_stubs[method_name] = true;
+        stub = Opal.stub_for(method_name);
 
-        if (!(method_name in subscriber)) {
-          subscriber[method_name] = stub;
+        for (j = 0; j < jlength; j++) {
```

Comment:
> When Opal runs Opal.add_stubs(['not_actually_a_method!']), OpalJQuery will already be registered as a subscriber by the require line at the top, right?

yes it would, the scenario I had in mind is this:

```rb
%x{
  var Foo = function() {}
  Foo.prototype.bar = function(){return 'bar'}
}

class Foo < `Foo`
  alias_native :bar
end

Object.new.baz rescue p $! # should be MethodMissing
Foo.new.baz rescue p $! # should be MethodMissing
```

where a new _subscriber_ (the bridged class) is added after the stub is created, but turns out it's not a problem, in fact testing the code above against this PR I get the correct behavior:

```
opal:pr/1624 ⤑ pbpaste | be bin/opal                                                                                                                                                                    ~/C/opal
#<NoMethodError: undefined method `baz' for #<Object:0x126>>
#<NoMethodError: undefined method `baz' for #<Foo:0x128>:Foo>
```